src/serializer/serializer.js

```diff
@@ -1748,6 +1833,6 @@ export class Serializer {
     let serialized = this.serialize(filename, code, sourceMaps);
     invariant(!this.logger.hasErrors());
     if (this.options.logStatistics) this.statistics.log();
+    serialized.generated.statistics = this.statistics;
```

Comment:
Cool, thats what I was doing, just wanted to make sure no one would get upset about the changed return structure. I updated the code to make the construction a little cleaner here, let me know if there are other things I should improve in this pr!